docs/mkdocs/docs/aws.md

```diff
@@ -0,0 +1,73 @@
+# Getting Started with AWS S3
+
+There are detailed guides on the AWS website on S3 configuration.  You can start with the [Creating your first bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html) page.
+
+Best practices with AWS S3 can depend on your situation.  If you're looking to try ArcticDB on AWS S3 then a reasonably simple approach is the following.
+
+### 1. Create an IAM user
+
+Best practice is to manage and access S3 buckets with a Non-root IAM account in AWS.   Ideally create a new account just for trialling ArcticDB.
+
+You can create an IAM account at [IAM > Users](https://console.aws.amazon.com/iamv2/home#/users).
+- Click `Add users`.
+- Come up with a username, e.g. `arcticdbtrial`, then click `Next`.
+- Select `Attach policies directly` and attach the `AmazonS3FullAccess` policy to that account, then click `Next`.
+- Click `Create user`.
+
+### 2. Create the access key
+
+- Click on the new user in the [IAM > Users](https://console.aws.amazon.com/iamv2/home#/users) table.
+- Go to `Security credentials` > `Access keys` and click `Create access Key`
+- Select the `Local code` option, ArcticDB is the local code.
+- Check `I understand...` and click `Next`.
+- Click `Create access Key`.
+- Record the Access key and Secret access key somewhere securely.
+
+### 3. Create the bucket
+
+The rest of the steps can be performed using commands on your client machine.  Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) if needed.
```

Comment:
I think that creating the bucket in the AWS GUI might be easier for novice users. Also, installing the CLI requires admin (Windows) / sudoers access, which might be hard to get on a corporate machine.

Reply:
Another option is to use boto3, users already need to be able to install python packages for ArcticDB, so it's a smaller expectation.